dbt/models/proximity/proximity.dist_pin_to_traffic_master.sql

```diff
@@ -0,0 +1,340 @@
+{{
+    config(
+        materialized='table',
+        partitioned_by=['year'],
+        bucketed_by=['pin10'],
+        bucket_count=1
+    )
+}}
+
+WITH traffic_minor AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Minor Arterial'
+),
+
+traffic_interstate AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Interstate'
+),
+
+traffic_freeway AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Freeway and Expressway'
+),
+
+traffic_local AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Local Road or Street'
+        AND daily_traffic IS NOT NULL
+),
+
+traffic_collector AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Major Collector'
+        OR road_type = 'Minor Collector'
+),
+
+traffic_other AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Other Principal Arterial'
+),
+
+distinct_pins AS (
+    SELECT DISTINCT
+        x_3435,
+        y_3435,
+        pin10
+    FROM {{ source('spatial', 'parcel') }}
+),
+
+distinct_years AS (
```

Comment:
This CTE seems to only get used once on line 769, where we grab the min year from it. We can probably just do that without the CTE.